charts/legalhold/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
nitpick: consistent delimiting in LOC:1. IMO there are need for them.

Reply:
I am not sure what you mean by this comment. Could you elaborate?

Reply:
I'm sorry, typos and wrong grammar.

Some of you yaml file start with `---` some don't. I suggested consistency, with the personal preference of removing it, since they are not needed. I hope, that makes the comment a bit more clear.

Reply:
Will do.